modules/shortcodes/slideshare.php

```diff
@@ -92,12 +92,12 @@ function slideshare_shortcode( $atts ) {
 	}
 
 	// check the margin width; if not empty, cast as int.
-	if ( ! empty( $attr['mw'] ) || '0' === $attr['mw'] ) {
+	if ( ( ! empty( $attr['mw'] ) || '0' === $attr['mw'] ) && ! Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
Could you check if the class exists before to use it, here and below? This may not be necessary  here in Jetpack but matters over on WordPress.com.

Reply:
@jeherve Thanks for your suggestion! This is already fixed in the last commit